tests/test_harvester_ice.py

```diff
@@ -0,0 +1,175 @@
+       should return values for both variables at once.
+       """
+    print("in test variable names")   
+    assert  'icetk' in  VALID_CONFIG_DICT['variable']
+
+def test_mean_values(tolerance=0.001):
+    """ 
+        The values of the calculated_means list were 
+        calculated from these eight forecast files:
+
+        bfg_1994010100_fhr09_snowiceocean_control.nc
+        bfg_1994010106_fhr06_snowiceocean_control.nc
+        bfg_1994010106_fhr09_snowiceocean_control.nc
+        bfg_1994010112_fhr06_snowiceocean_control.nc
+        bfg_1994010112_fhr09_snowiceocean_control.nc
+        bfg_1994010118_fhr06_snowiceocean_control.nc
+        bfg_1994010118_fhr09_snowiceocean_control.nc
+        bfg_1994010200_fhr06_snowiceocean_control.nc
+        
+        When averaged together, these files represent a 24 hour mean. The 
+        average values hard-coded in this test was calculated from 
+        forecast files using a separate python code.
+    """
+    data1 = harvest(VALID_CONFIG_DICT)
+    for item in data1:
+        if item.statistic == 'mean':
+           calculated_means = [0.14929623501925735,0.05702953727861002] 
+           for index in range(len(calculated_means)):
+               assert calculated_means[index] <= (1 + tolerance) * item.value[index]
+               assert calculated_means[index] >= (1 - tolerance) * item.value[index]
+
+def test_gridcell_variance(tolerance=0.001):
+    """
+      The values of the calculated_variances list were calculated
+      from the forecast files listed above in a separate python script.
+      """
+    data1 = harvest(VALID_CONFIG_DICT)
+       
+    for item in data1:
+        if item.statistic == 'variance':
+          calculated_variances = [0.277846268686431,0.05878632288856643]
+          for index in range(len(calculated_variances)):
+              assert calculated_variances[index] <= (1 + tolerance) * item.value[index]
+              assert calculated_variances[index] >= (1 - tolerance) * item.value[index]
+  
+def test_gridcell_min_max(tolerance=0.001):
+    data1 = harvest(VALID_CONFIG_DICT)
+    
+    for item in data1:
+        if item.statistic == 'minimum':
+           calculated_min  = [5.967400007991776e-06,8.364692032651244e-06]
+           for index in range(len(calculated_min)):
+               assert calculated_min[index] <= (1 + tolerance) * item.value[index]
+               assert calculated_min[index] >= (1 - tolerance) * item.value[index]
+
+        if item.statistic == 'maximum':
+           calculated_max = [5.368260566834775,3.4056593791028917]
+           for index in range(len(calculated_max)):
+               assert calculated_max[index] <= (1 + tolerance) * item.value[index]
+               assert calculated_max[index] >= (1 - tolerance) * item.value[index]
+
+    
+def test_units():
+    variable_dictionary = {}
+    data1 = harvest(VALID_CONFIG_DICT)
+
+    for item in data1:
+        if item.variable == 'snod':
```

Comment:
Two things came from this. One, I don't think this code is ever being hit because the harvest config dictionary doesn't request them. Two, this brought to mind the fact that none of these tests are effectively checking for the variables. I think what would be better is to search the items for the variable via name first to see if it actually gets returned, that way it can error out if it isn't there and then we can identify problems that way and make sure this code is actually being run.